install-mma-arc-extension/arm template and bicep/README.md

```diff
@@ -1,31 +1,31 @@
 # AzSHCI-Automation
 ARM template to install Azure Windows Monitoring Agent Arc Extension for Azure Stack HCI Clusters.
 	1. Creates DCR rule 
-	2. Installs AMA on clusters 
+	2. Installs MMA on clusters 
 	3. associates the DCR rule with the extension
 
 
 Pre-requisites:
 	1. Create Workspace 'microsoft.operationalinsights/workspaces' manually before running script.
 	2. Install Az Powershell https://learn.microsoft.com/en-us/powershell/azure/install-azure-powershell?view=azps-9.6.0
 
-Fill up the values in ama-parameters.json: 
+Fill up the values in mma-parameters.json: 
 
-1. cluster-names: array of the clusters to install AMA on 
+1. cluster-names: array of the clusters to install MMA on 
 2. location: location where to create the DCR rule 
 3. Workspace-id, workspace-name: details of the workspace created above
 
 
 To run the ARM-template: 
 
-New-AzResourceGroupDeployment -Name <deployment-name> -ResourceGroupName <resource-group> -TemplateFile .\install-ama.json -TemplateParameterFile .\ama-parameters.json
+New-AzResourceGroupDeployment -Name <deployment-name> -ResourceGroupName <resource-group> -TemplateFile .\mma_template.json -TemplateParameterFile .\mma_parameters.json
```

Comment:
the file name is install_mma.json, no?

Reply:
In the folder structure template file name is mma_template.json.